### PR TITLE
Custome Service IDP and Chatbot deployment in production

### PR DESCRIPTION
**Pull request by Sh3rd3n**

### CODE REVIEW

File: ionos_prd/namespaces/chatbot-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
```
Comment (bialask-daiteap) [inline ionos_prd/namespaces/chatbot-backend.yaml:3]: Namespaces for PRD are generically defined in folder ionos_common/namespaces
Please add resources there and rebase the changes

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

**Comment by Sh3rd3n** [comment]
Ok, tell me if its correct now

File: ionos_prd/namespaces/chatbot-backend.yaml (hunk 1 above)
Comment (bialask-daiteap) [reply ionos_prd/namespaces/chatbot-backend.yaml:3]: Please remove that one. The comment above goes for cs-identity namespace too

Comment (bialask-daiteap) [reply ionos_prd/namespaces/chatbot-backend.yaml:3]: also , please rebase the changes and squash the commits

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

**Comment by Sh3rd3n** [comment]
I don't know hot to rebase and squash on my git client, i just close this PR and open a new one.

**Pull request closed without merging**